src/DropDownMenu/DropDownMenu.js

```diff
@@ -364,6 +370,7 @@ class DropDownMenu extends Component {
             {displayValue}
           </div>
           <IconButton
+            disableTouchRipple={disableTouchRipple === undefined ? disabled : disableTouchRipple}
```

Comment:
The passing tests are suggesting that the issue is coming from somewhere else. And true, from what I can see the true issue is that we don't forward the `disabled` property to the IconButton.
That piece of logic is already inside the `EnhancedButton` component. No need for duplication.